### ISSUE TITLE
Publishing / SendToQueue On a Cluster

### ISSUE THREAD

**Issue by CraigHarley** [benbria/node-amqp-connection-manager#28]
I've got it set up and publishing to the cluster. If a node dies the target channel changes, which is great.

However I'd like to spread the load across multiple nodes since I'm firing a ton of messages, how can I do this?

I'd expect to be able to pass an argument to `sendToQueue` to signal which node to send the message to, or is what I'm after not possible with this plugin?

**Comment by jwalton** [comment]
No, we don't do anything that fancy.  `ampq-connection-manager` only connects to one of the nodes in the cluster at a time.  If the connection dies, it cycles to the next one, but it never has more than one connection open at at time.

If you're interested in submitting a PR to allow being connected to multiple servers, I'd be game.  I'll even convert this whole project from coffee-script to JS for you if you want to.  :)

**Comment by CraigHarley** [comment]
I have something like that working from a user side, I'll look at baking it into a PR over the weekend. 

Yeah it's been a long time since I've written any coffee script.

The API I would have in mind would be setting a flag on `createChannel`, something like `balanceLoad` which would balance the publishes across the array passed to `connect`.
